pkg/nodes/node.go

```diff
@@ -189,6 +219,32 @@ func (builder *NodeBuilder) ExternalIPv4Network() (string, error) {
 	return extNetwork.IPv4, nil
 }
 
+// WaitUntilNodeStable waits for the duration of the defined timeout or until the
+// node gets to a stable - not unschedulable.
+func (builder *NodeBuilder) WaitUntilNodeStable(timeout time.Duration) error {
```

Comment:
I think a more appropriate name for this method is `WaitUntilSchedulable`. node is the name of the pkg so no need to add it to the method name.

Reply:
You can then update subsequent logs/comments from `not unschedulable` to just `schedulable`